assets/src/block-templates/quick-links/template.js

```diff
@@ -23,7 +23,7 @@ const template = ({
   backgroundColor = 'grey-05',
 }) => ([
   ['core/group', {
-    className: `block ${metadata.classname}`,
+    className: 'block',
```

Comment:
This `metadata.classname` resulted in `undefined`, and it seems that we don't need it